pymongo_voyageai/storage.py

```diff
@@ -14,6 +14,9 @@ class ObjectStorage:
     root_location: str
     """The root location to use in the object store."""
 
+    url_prefixes: list[str] | None
```

Comment:
typo in docstring
"""A class used store image documents.""" 
"""A class used to store image documents."""